src/conn/conn_tiered.c

```diff
@@ -368,28 +381,35 @@ __wt_flush_tier(WT_SESSION_IMPL *session, const char *config)
      * We have to hold the lock around both the wait call for a previous flush tier and the
      * execution of the current flush tier call.
      */
+    locked = false;
```

Comment:
Do we need to set `locked = false`?  I don't see anything that would look at its value before it gets set to `true` after acquiring the lock.